src/derivative_wrappers.jl

```diff
@@ -220,15 +220,15 @@ function jacobianmat!(JM::AbstractMatrix{<:Number}, f, x::AbstractArray{<:Number
 end
 function vecjacobian!(dλ, y, λ, p, t, S::TS;
                       dgrad = nothing, dy = nothing,
-                      W = nothing) where {TS <: SensitivityFunction}
-    _vecjacobian!(dλ, y, λ, p, t, S, S.sensealg.autojacvec, dgrad, dy, W)
+                      W = nothing, kwargs...) where {TS <: SensitivityFunction}
```

Comment:
we should probably be more specific than a full splat

Reply:
How would I do that? By adding the `recompile_tape` kwarg to all `_vecjacobian!` functions even if it's not used except in ReverseDiffVJP?